make gosdk compilable to wasm (go-ethereum not compatible with wasm)

@moldis please fix this since go-ethereum package is your responsibility.

You can reproduce the compile error by doing this:
1. git checkout jssdk branch
2. cd $TOP/_sdkver
3. GOOS=js GOARCH=wasm go build -o proxy.wasm proxy.go ethwallet.go
4. produces this error message:

```
../../../go/pkg/mod/github.com/ethereum/go-ethereum@v1.10.3/metrics/cpu_enabled.go:40:79: undefined: "github.com/shirou/gopsutil/cpu".ClocksPerSec
../../../go/pkg/mod/github.com/ethereum/go-ethereum@v1.10.3/metrics/cpu_enabled.go:41:47: undefined: "github.com/shirou/gopsutil/cpu".ClocksPerSec
../../../go/pkg/mod/github.com/ethereum/go-ethereum@v1.10.3/metrics/cpu_syscall.go:30:12: undefined: syscall.Getrusage
../../../go/pkg/mod/github.com/ethereum/go-ethereum@v1.10.3/metrics/cpu_syscall.go:30:30: undefined: syscall.RUSAGE_SELF
```

The temporary workaround that I'm using (in order to build the wasm taht Hau can use) is to edit those go-ethereum files locally and fix the golang errors by nuking the CPU stat collection.

In order to fix this, Artem you need to do either one of the following two:
* submit patch to go-ethereum to fix their build for wasm by nuking the CPU stats
* fork go-ethereum, host a public version on 0chain with the fixed patch that wasm can build.

@moldis  also this is way outside the scope of my responsibilities, so I'm assigning to you

--- a/go.mod
+++ b/go.mod
@@ -1,11 +1,16 @@
 module github.com/0chain/gosdk
 
 require (
-	github.com/ethereum/go-ethereum v1.10.3
+	github.com/aristanetworks/goarista v0.0.0-20190912214011-b54698eaaca6 // indirect
+	github.com/elastic/gosigar v0.10.5 // indirect
+	github.com/ethereum/go-ethereum v1.10.4
 	github.com/h2non/filetype v1.0.9
+	github.com/herumi/bls-go-binary v1.0.0
 	github.com/klauspost/reedsolomon v1.9.11
-	github.com/miguelmota/go-ethereum-hdwallet v0.0.0-20210314074952-8dd49aa599b9
+	github.com/miguelmota/go-ethereum-hdwallet v0.1.0
 	github.com/mitchellh/mapstructure v1.1.2
+	github.com/steakknife/bloomfilter v0.0.0-20180922174646-6819c0d2a570 // indirect
+	github.com/steakknife/hamming v0.0.0-20180906055917-c99c65617cd3 // indirect
 	github.com/stretchr/testify v1.7.0
 	github.com/tyler-smith/go-bip39 v1.1.0
 	go.dedis.ch/kyber/v3 v3.0.5
@@ -15,3 +20,5 @@ require (
 )
 
 go 1.13
+
+replace github.com/ethereum/go-ethereum => github.com/certifaction/go-ethereum v1.10.3-wasm

